ios/ReactNativePageView.m

```diff
@@ -472,6 +473,17 @@ - (void)shouldShowPageIndicator:(BOOL)showPageIndicator {
     if (_reactPageIndicatorView){
         _reactPageIndicatorView.hidden = !showPageIndicator;
     }
+    
+    // https://github.com/react-native-community/react-native-viewpager/issues/182
+    if (_transitionStyle == UIPageViewControllerTransitionStylePageCurl) {
+        NSArray *subviews = self.reactPageViewController.view.subviews;
+        for (int i=0; i<[subviews count]; i++) {
+            if ([[subviews objectAtIndex:i] isKindOfClass:[UIPageControl class]]) {
```

Comment:
You can use `subviews[i]` instead of `[subviews objectAtIndex:i]` (the same also below).

Reply:
I used https://github.com/react-native-community/react-native-viewpager/pull/184#discussion_r442012212
hence I don't need to use index anymore :D